### ISSUE TITLE
My Applications Page Design

### ISSUE THREAD

**Issue by shiraazAh** [shiraazAh/osmium-job-app#49]
Once the Application card is finished, Just create the page similar to this design: https://www.figma.com/design/irAe81EmWNtlsH9TAFwMxA/Osmium-App?node-id=171-156&node-type=frame&t=xyOD87ArYJLFzLAa-0

If the Card is finished, you would just have to display the cards and add the Page heading and nothing else - There is another card for API Implementation
